components/dashboard/src/user-settings/Preferences.tsx

```diff
@@ -72,6 +89,49 @@ export default function Preferences() {
                         </p>
                     </div>
                 </div>
+
+                <h3 className="mt-12">Inactivity Timeout </h3>
+                <p className="text-base text-gray-500 dark:text-gray-400">
+                    By default, workspaces stop following 30 minutes without user input (e.g. keystrokes or terminal
+                    input commands). You can increase the workspace timeout up to a maximum of 24 hours.
+                </p>
+                <div className="mt-4 max-w-xl">
+                    <h4>Default Workspace Inactivity Timeout</h4>
+
+                    {!allowConfigureWorkspaceTimeout && (
+                        <Alert type="message">
+                            Only paid users can update the inactivity timeout. See
+                            <a href="https://www.gitpod.io/pricing" target="price-info" className="gp-link">
+                                {" "}
+                                pricing{" "}
+                            </a>
+                            for more.
```

Comment:
From what I understand, **yes**.

If the free user can not edit the input, there's no need to show the input disabled. Generally, disabled buttons or disabled inputs are not best for accessibility.

However, this can easily come back to us once we split workspace usage across orgs. If a user is on an org with a active billing plan, do we let them have a custom workspace timeout for workspaces in other orgs? Probably nothing to worry about now, but worth highlighting, right?